### ISSUE TITLE
Feature Request: Highlight Test Coverage

### ISSUE THREAD

**Issue by gordoncyu** [jeapostrophe/racket-langserver#134]
It would be great if other editors could use the language server to highlight code not covered by tests, just like how DrRacket does.

**Comment by dannypsnl** [comment]
Do you mean the functionality describe in https://docs.racket-lang.org/plait/testing-tutorial.html?

**Comment by gordoncyu** [comment]
Yes, I am referring to the "Syntactic test suite coverage". Currently I am just running `raco cover thing.rhm` and opening the html file in chrome, but giving this information in editor would definitely smooth my workflow and be aligned with the stated goal of "provid[ing] functionality that mimics DrRacket's code tools as closely as possible" in the README, helping in the way any LSP should.

**Comment by 6cdh** [comment]
I think lsp server can't do that. Test coverage requires to run the code, while lsp server does not run the code and should not run.

There should be a editor specific plugin to do it. For example, there is a button or codelens, click it to run tests in repl, and repl does something under the hood, then show information in editor.

**Comment by dannypsnl** [comment]
Make sense, @gordoncyu you can also open this request to https://github.com/Eugleo/magic-racket.

**Comment by gordoncyu** [comment]
I use neovim btw, so doing so would not be my prerogative. I followed @6cdh's advice, writing my own plugin: https://github.com/gordoncyu/racket-cover.nvim.
